.github/workflows/uberjar.yml

```diff
@@ -2,18 +2,25 @@ name: Uberjar
 
 on:
   push:
-    branches:
-      - '**'
+    branches-ignore:
```

Comment:
I think the other jobs in this workflow should still be executed for all branches.
If you want to special-case/ignore certain branch for `containerize` job, better put a branch conditional (`if`) under that job.

Reply:
ready to go